django/utils/datastructures.py

```diff
@@ -131,18 +131,21 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
-        """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
-        """
+    def _getlist(self, key, default=None):
```

Comment:
If `MultiValueDict.getlist` is supposed to be immutable then change should be made there. `QueryDict` clearly states that it's immutable, but couldn't' find such an indication in `MultiValueDict`.